language_server/src/lsp/lsp_server.cpp

```diff
@@ -90,7 +90,7 @@ void server::message_received(const json& message)
     auto params_found = message.find("params");
     auto method_found = message.find("method");
 
-    if (params_found == message.end() || method_found == message.end())
+    if (method_found == message.end())
     {
         LOG_WARNING("Method or params missing from received request or notification");
         send_telemetry_error("lsp_server/method_or_params_missing");
```

Comment:
Probably doesn't need to be changed so just noting that it's not entirely accurate right now:

```suggestion
        send_telemetry_error("lsp_server/method_missing");
```